src/vulkan/pipeline.h

```diff
@@ -80,6 +92,10 @@ class Pipeline {
 
   PipelineType pipeline_type_;
   std::vector<std::unique_ptr<Descriptor>> descriptors_;
+
+  uint32_t fence_timeout_ms_ = 100;
+
+  std::vector<VkPipelineShaderStageCreateInfo> shader_stage_info_;
```

Comment:
nit: move this up to line 95 and remove the blank lines. Groups the vectors together.

Reply:
Done.